third_party/silabs/silabs_board.gni

```diff
@@ -165,6 +162,9 @@ declare_args() {
   # User can set this arg to false to skip the rps creation for WiFi SoCs.
   # e.g. for CI
   use_rps_extension = wifi_soc
+
+  # UART log forwarding enabled by default for 917 SoC
+  sl_uart_log_output = wifi_soc
```

Comment:
From an outsider's perspective what does `wifi_soc` have to do with default for 917 SoC? should this not just be set to true based on the comment? Could you not set it to false near the top in a declare_args, but set it to true for the 917?

Reply:
updated the comment, in the context wifi_soc is nothing but the 917 soc both are same. Updated the comment to say
`# Default true for the wifi soc and false for EFR32 boards`